api/WorkerGlobalScope.json

```diff
@@ -666,7 +666,7 @@
               "version_added": "4.0"
             },
             "webview_android": {
-              "version_added": true
+              "version_added": false
```

Comment:
Why false here? It’s in Chrome for Android...

Reply:
Looks like this is a case of a bad test in the collector (I'm surprised that the rest of the tests worked properly).  I'll have a fix for this in collector v1.1.9; reverting and adding version numbers.

Reply:
Great, thanks!